src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random;
+
+    public ColorSupplier(Random random) {
+        this.random = random;
+    }
+
     public String getRandomColor() {
-        return null;
+        int indexOfConstant = random.nextInt(Colors.values().length);
```

Comment:
let's rename `indexOfConstant` to more explanation name, e.g. `colorIndex` or `randomColorIndex`